portal-web/test/functional/com/liferay/portalweb/macros/ChangeList.macro

```diff
@@ -147,6 +147,16 @@ definition {
 			value1 = "Revert &quot;${publicationName}&quot;");
 	}
 
+	macro searchCP {
```

Comment:
I'm not sure if there will be an equivalent "PG" macro. If there won't be and we don't need this distinction, this macro can just be "search".

Reply:
There is no PG macro, and I update the macro to search, thank you for your suggestion.